### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by Jayman2000** [Yamakaky/joy#5]
I want to create a game that uses the Ring-Con. Could you please place this repo under an open-source license?

**Comment by Yamakaky** [comment]
Hey, nice to see these crates get some use! I haven't thought about which license, what would be the license of your game?

**Comment by Jayman2000** [comment]
I’ll probably end up using the MIT license. The engine I’ll be using ([Godot](https://godotengine.org/)) is under the MIT license.

**Comment by Yamakaky** [comment]
Done! Please keep me updated on your game!

**Comment by Jayman2000** [comment]
> Done! Please keep me updated on your game!

Alright, I have my first update. When I opened this issue, I didn’t know Rust at all. Since then, I’ve been learning Rust by going through parts of [the book](https://doc.rust-lang.org/stable/book/) and working on projects unrelated to the game. One of those projects was [a very amateurish statically generated Web site](https://jasonyundt.neocities.org/). (The static site generator ended up using very little Rust code in the end, but I still learned a lot).

I really appreciate the fact that you asked me to keep you updated. That’s what motivated me to create a Web site. I want to have a central place to post updates for anyone who was interested.

I wrote [a blog post that details how chose joy in particular](https://jasonyundt.neocities.org/posts/finding-the-right-ringcon-api.html) (as opposed to any of the other pieces of software that can read Joy-Con/Ring-Con inputs). I hope to have more updates soon!
